src/translate.service.ts

```diff
@@ -396,13 +396,12 @@ export class TranslateService {
      * @returns string
      */
     public getBrowserLang(): string {
-        if(typeof window === 'undefined' || typeof window.navigator === 'undefined') {
+        let browserLang = this.getBrowserCultureLang();
+
+        if(browserLang === 'undefined') {
```

Comment:
I was referring to the browserLang check, it doesn't use `typeof`. So probably it should be changed to

```js
if(typeof browserLang === 'undefined') {
```